src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+### The Evolution of Logic
+{ /* cspell: disable-next-line */ }
+> [S]ince Aristotle, [logic] has taken no backward step, but also none forward, the latter because to all appearances it seems to be finished and complete (Hegel, 2010, 30/21.35).
+
+While Hegel gives a dismal appraisal of the history of logic since Aristotle, the issue is as much a philosophical as a social and historical one. The philosophical issue being that logic is regarded as something inherently complete and static. "Its determinations," Hegel writes, "are accepted in their undisturbed fixity and are brought together only in external connection" (Hegel, 2010, 32/21.36). Logic, as it is typically known outside a philosophical context, amounts to something of an accounting of various logical forms which does not critically assess their immanent connections or attempt to derive a systematic deduction of the forms themselves. While Hegel speaks here from his own time about the state of logic, the underlying philosophical assumption is that the _method_ of logic is itself constant and perfectly concluded. 
+
+Curiously, Hegel specifically complains that in his present situation that there were no trace of the scientific method in the typical considerations of logic. While the scientific method has come to include notions like observations, question, hypothesis, prediction, experimentation, reproduction, conclusion from established observation and communication centered around empirical objects, the general idea is not far off from what Hegel has in mind here, namely, that the form of logic itself is actively failing to live up to the standards of scientific inquiry. Logic itself and its forms (propositions, inference, categories, etc.) are not being scientifically examined in accordance with rigorous questioning, experimentation, reproduction and communication.[^3] Indeed, Hegel points to empirical science and pure mathematics as having their own suitable methods whereas this is sorely lacking in the domain of logic. This has led to philosophy borrowing methods from these other domains and attempting to apply them to logic, but Hegel thinks that this is an error. Yet, it is equally an error to crudely reject any method whatsoever.
```

Comment:
"but Hegel thinks that this is an error" - might be worth hammering home why he think's it's an error, i.e. because it is an externally applied method.